helm/conjur-config-cluster-prep/templates/tests/_helm_test.bats.txt

```diff
@@ -50,7 +50,7 @@ function conjur_access_token_exists() {
   run conjur_is_reachable
   if [ "$status" -ne 0 ]; then
     display_error "The 'conjur.applianceURL' chart value is set to\n" \
```

Comment:
Thanks for cleaning up line 53. If you're making any other changes, the `applianceURL` should also be changed to `applianceUrl` on line 52.

Reply:
Changed line 52 also.